### DISCUSSION TITLE
expires_in key for cache

### DISCUSSION THREAD

**Discussion by Stanislav-Lapata** [ElMassimo/oj_serializers#20] [Q&A]
In our project we have explosive load on the server but it usually take few minutes and I need to cache date from 1 to 10 minutes.

`Rails.cache.fetch` and `ActiveSupport::Cache::MemoryStore.new.fetch` can accept [`expires_in`](https://api.rubyonrails.org/classes/ActiveSupport/Cache/Store.html#method-i-fetch) key as the second arg
for example 
```ruby
Rails.cache.fetch("some_key", expires_in: 1.minute) do
  # some code
end

cache = ActiveSupport::Cache::MemoryStore.new
cache.fetch("some_key", expires_in: 1.minute) do
  # some code
end
```

As I understand right I can't use `expires_in` key right now
https://github.com/ElMassimo/oj_serializers/blob/v2.0.3/lib/oj_serializers/serializer.rb#L250-L251

Is it possible to add new method that will extend the second arg or add new arg to cached/cached_with_key? 

Thanks

**Comment by dimroc** [comment]
I also need expires_in for caching. Any chance we could incorporate/revisit this?
